src/pages/cluster-management/clusterInfo.html

```diff
@@ -0,0 +1,135 @@
+<link href="css/clustermanagement.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
+<title>{{'view.clusterManagement.title' | translate}}</title>
+
+<h1>
+    {{title}}
+    <span class="btn btn-link"
+          ng-class="{ 'help-button': showExternalElements, 'help-button-disabled': !showExternalElements }"
+          popover-template="'js/angular/templates/titlePopoverTemplate.html'"
+          popover-trigger="mouseenter"
+          popover-placement="bottom"
+          popover-append-to-body="true"><span class="icon-info"></span></span>
+</h1>
+
+<div onto-loader-new
+     ng-show="loader"
+     class="cluster-loader"
+     message="getLoaderMessage"
+     size="75">
+</div>
+
+<div class="clearfix mb-2">
+    <button type="button" class="btn btn-primary create-cluster-btn" ng-disabled="clusterConfiguration"
+            ng-click="createCluster()"
+            tooltip="{{'cluster_management.buttons.create_cluster_tooltip' | translate}}" tooltip-placement="top">
+        <span class="icon-plus"></span> {{'cluster_management.buttons.create_cluster' | translate}}
+    </button>
+</div>
+
+<div class="ng-scope" ng-hide="clusterConfiguration">
+    <p class="alert alert-danger">
+        {{'cluster_management.no_cluster_configured' | translate}}
+    </p>
+</div>
+
+<div ng-if="clusterConfiguration">
+    <div class="row" ng-if="clusterConfiguration">
```

Comment:
Do you need the double `clusterConfiguration` check?

Reply:
No, removed it.